docs/requirements.txt

```
@@ -0,0 +1,2 @@
+Sphinx
+sphinx-book-theme
```

```suggestion
Sphinx==3.5.3
sphinx-book-theme==0.1.0
```
I would suggest fixing the versions. The suggested versions are because the built documentation will be consistent with the current one, but it's open to discussion if there is a need to use newer versions.